tasks.py

```diff
@@ -399,6 +399,11 @@ def database_exists():
     else:
         return True
 
+def trial():
```

Comment:
I think you should write a docstring here so that we can understand what this method does